Tests/TestFrameworks/TUnit.Specs/FrameworkSpecs.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using FluentAssertions;
+using TUnit.Assertions.Exceptions;
```

Comment:
The test should not directly reference `TUnit`.

This recently changed in #2712, but is being reverted as part of #2718.

See https://github.com/fluentassertions/fluentassertions/pull/2718#discussion_r1720934926

Reply:
Interesting. By removing the reference it did indeed remove my assertion assembly from the app domain. So I had to do the similar try/catch assembly load approach that xUnit 2 was doing as well.